tests/Json.Tests.ps1

```diff
@@ -643,4 +643,249 @@ Describe 'Module' {
             }
         }
     }
+
+    Context 'Export-Json' {
+        BeforeAll {
```

Comment:
Add a test that takes an array and outputs as a json structure where the outer layer is an array "[ ... ]". This is expected behavior for (all) json functions.